core/src/main/java/com/orange/ouds/core/theme/OudsColorScheme.kt

```diff
@@ -496,13 +664,13 @@ fun OudsColorScheme.fromToken(token: OudsColorKeyToken.Always): Color {
 }
 
 @Stable
-fun OudsColorScheme.fromToken(token: OudsColorKeyToken.Background): Color {
+fun OudsColorScheme.fromToken(token: Background): Color {
```

Comment:
`import com.orange.ouds.theme.tokens.OudsColorKeyToken.Background` should be removed in order to use the full name `OudsColorKeyToken.Background` like what we do with other types.